js/msp/msp_helperfunctions.js

```diff
@@ -0,0 +1,1813 @@
+'use strict';
+MSP.listen(MSP_process_data);
```

Comment:
Adding the listener on load of the JS file is not a good idea. I'd rather do it in main.js, as part of the setup of MSP, or, once the MSP translation layer is turned into a prototype, in its constructor.